README.md

```diff
@@ -42,7 +42,7 @@ You will also need to install:
       * You also need to install the `Command Line Tools` via Xcode. You can find this under the menu `Xcode -> Preferences -> Downloads`
       * This step will install `gcc` and the related toolchain containing `make`
   * On Windows:
-    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator).
+    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator). Note: although windows-build-tools will install Python, it will not add python to your PATH, so you will have to add it manually.
```

Comment:
Might be worth noting this PS command to add Python to your path (change `c:\python27` to wherever you installed Python):

```powershell
[System.Environment]::SetEnvironmentVariable('Path', "c:\python27;$env:Path", "Process")
```

The last param can be `Machine | User | Process`, but if you use only Machine or User it won't take effect till you relaunch PS or CMD sessions.

/cc @felixrieseberg - maybe you can add this to the scripts?